### PR TITLE
Changes for 8.16

### PR DESCRIPTION
**Pull request by mattam82**
Comments welcome!

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+Georges Gonthier, Benjamin Grégoire, Jason Gross, Hugo Herbelin,
+Vincent Laporte, Olivier Laurent, Assia Mahboubi, Kenji Maillard,
+Guillaume Melquiond, Pierre-Marie Pédrot, Clément Pit-Claudel, Pierre Roux,
+Kazuhiko Sakaguchi, Vincent Semeria, Michael Soegtrop, Arnaud Spiwack,
+Matthieu Sozeau, Enrico Tassi, Laurent Théry, Anton Trunov, Li-yao Xia
+and Théo Zimmermann. See the `Coq Team face book <https://coq.inria.fr/coq-team.html>`_
+page for more details.
+
+The 57 contributors to the 8.16 versions are Tanaka Akira, Frédéric Besson, Martin Bodin, Ana Borges,
+Ali Caglayan, Minki Cho, Cyril Cohen, Juan Conejero, "stop-cran", Adrian Dapprich, Maxime Dénès,
```
Comment (jfehrle) [inline doc/sphinx/changes.rst:81]: @stop-cran Would you like us to include your real name here?

**Review by jfehrle — COMMENTED** [review]
wording suggestions

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.16 integrates changes to the Coq kernel, performance improvements along
+with a few new features. We highlight some of the most impactful changes here:
+
+  - The guard checker now ensures strong :ref:`normalization <816Normalization>`
+    under any reduction strategy.
+
+  - Irrelevant terms (in the strict propositions sort) are now squashed to a dummy
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/changes.rst:23]: Right

File: doc/sphinx/changes.rst [hunk 3]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+    value during :ref:`conversion <816SPropConversion>`, fixing a subject reduction issue and making proof
+    conversion cheap.
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/changes.rst:25]: Faster is better indeed.

File: doc/sphinx/changes.rst [hunk 4]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
 ------------
 
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.16 integrates changes to the Coq kernel, performance improvements along
+with a few new features. We highlight some of the most impactful changes here:
+
+  - The guard checker now ensures strong :ref:`normalization <816Normalization>`
+    under any reduction strategy.
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply doc/sphinx/changes.rst:21]: Documentation for the guard checker should probably be added somewhere close to https://coq.inria.fr/refman/language/core/inductive.html#coq:cmd.Fixpoint.

File: doc/sphinx/changes.rst [hunk 5]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+  - Standard library `reorganization and deprecations <816Stdlib>`.
+
+  - Improve the treatment of standard library numbers by ``Extraction``.
+
+See the `Changes in 8.16.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 8.16 <https://coq.github.io/doc/v8.16/refman>`_,
+`documentation of the 8.16 standard library <https://coq.github.io/doc/v8.16/stdlib>`_
+and `developer documentation of the 8.16 ML API <https://coq.github.io/doc/v8.16/api>`_
+are also available.
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/changes.rst:51]: The links change with each revision, so I think it still makes sense.

File: doc/sphinx/changes.rst [hunk 6]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+Paolo G. Giarrusso, Attila Gáspár, Gaëtan Gilbert, Rudi Grinberg, Jason Gross, Hugo Herbelin,
+Wolf Honore, Jasper Hugunin, Bart Jacobs, Emilio Jesús Gallego Arias, Pierre Jouvelot,
+Ralf Jung, Grant Jurgensen, Jan-Oliver Kaiser, Wojciech Karpiel, Thomas Klausner,
+Ethan Kuefner, Fabian Kunze, Olivier Laurent, Yishuai Li, Erik Martin-Dorel, Guillaume Melquiond,
+Jean-Francois Monin, Pierre-Marie Pédrot, Rudy Peterson, Clément Pit-Claudel, Seth Poulsen,
+Ramkumar Ramachandra, Pierre Roux, Takafumi Saikawa, Kazuhiko Sakaguchi, Gabriel Scherer,
+Vincent Semeria, Kartik Singhal, Michael Soegtrop, Matthieu Sozeau, Enrico Tassi, Laurent Théry,
+Anton Trunov, Li-yao Xia and Théo Zimmermann.
+
+The Coq community at large helped improve the design of this new version via
```
[the remark this answers is not in the document]
Comment (Zimmi48) [reply doc/sphinx/changes.rst]: Was this comment meant for this thread? I'm lacking context here.

File: doc/sphinx/changes.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (Alizter) [reply doc/sphinx/changes.rst:51]: Regarding the version selector: https://github.com/coq/coq/issues/14983

File: doc/sphinx/changes.rst [hunk 7]
```diff
@@ -20,6 +110,8 @@ Changes in 8.16.0
 Kernel
 ^^^^^^
 
+ .. _816Normalization
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:113]:
```suggestion
 .. _816Normalization:
```

I guess that this is what makes the build fail.

File: doc/sphinx/changes.rst [hunk 8]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+    structures. Also allow coercions that do not :ref:`fullfill the uniform inheritance condition <816UniformInh>`.
+
+  - :ref:`Generalized rewriting <816GeneralizeRew>` support for rewriting with ``Type``-valued relations and in
+    ``Type`` contexts, using the ``Classes.CMorphisms`` library.
+
+  - New :ref:`boolean equality <816BooleanEquality>` scheme command for decidable inductive types.
+
+  - New :ref:`Print Notation <816PrintNotation>` command.
+
+  - Incompatibilities in :ref:`name generation <816ProgramObls>`` for Program obligations,
```
[the remark this answers is not in the document]
Comment (mattam82) [reply doc/sphinx/changes.rst:38]: Yes, we introduced incompatibilities

File: doc/sphinx/changes.rst [hunk 9]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+  - Introduction of :ref:`reversible coercions <816ReversibleCoercions>`, which
+    allow coercions relying on meta-level resolution such as type-classes or canonical
+    structures. Also allow coercions that do not :ref:`fullfill the uniform inheritance condition <816UniformInh>`.
+
+  - :ref:`Generalized rewriting <816GeneralizeRew>` support for rewriting with ``Type``-valued relations and in
+    ``Type`` contexts, using the ``Classes.CMorphisms`` library.
+
+  - Added the :ref:`boolean equality <816BooleanEquality>` scheme command for decidable inductive types.
+
+  - Added a :ref:`Print Notation <816PrintNotation>` command.
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:36]: Apparently, you forgot to define this label.

File: doc/sphinx/changes.rst [hunk 10]
```diff
@@ -249,9 +358,11 @@ SSReflect
 Commands and options
 ^^^^^^^^^^^^^^^^^^^^
 
+ .. _816ModuleExprs:
+
 - **Changed:**
   :cmd:`Module` now only allows parentheses around module arguments. For instance, ``Module M := (F X).`` is now a parsing error
-  (`#15355 <https://github.com/coq/coq/pull/15355>`_,
+  (`#15355 <https://github.com/coq/coq/pull/15355>`_,s
```
Comment (proux01) [inline doc/sphinx/changes.rst:365]:
```suggestion
  (`#15355 <https://github.com/coq/coq/pull/15355>`_,
```

File: doc/sphinx/changes.rst [hunk 11]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+Ethan Kuefner, Fabian Kunze, Olivier Laurent, Yishuai Li, Erik Martin-Dorel, Guillaume Melquiond,
+Jean-Francois Monin, Pierre-Marie Pédrot, Rudy Peterson, Clément Pit-Claudel, Seth Poulsen,
+Ramkumar Ramachandra, Pierre Roux, Takafumi Saikawa, Kazuhiko Sakaguchi, Gabriel Scherer,
+Vincent Semeria, Kartik Singhal, Michael Soegtrop, Matthieu Sozeau, Enrico Tassi, Laurent Théry,
+Anton Trunov, Li-yao Xia and Théo Zimmermann.
+
+The Coq community at large helped improve this new version via
+the GitHub issue and pull request system, the Coq development mailing list
+coqdev@inria.fr, the coq-club@inria.fr mailing list, the `Discourse forum
+<https://coq.discourse.group/>`_ and the `Coq Zulip chat <https://coq.zulipchat.com>`_.
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:95]: I think it's time to remove coqdev from this list. The last discussions on this mailing list are from May 2021.

**Comment by Zimmi48** [comment]
There was an issue with the issue counting in coqbot that was fixed a few months ago but affected the counting for the 8.15 and 8.16 releases. Here are the fixed counts:
- 8.16.0 (the count was correct but has increased in the last few days): 6 closed issues
- 8.16+rc1: 93
- 8.15.2: 10
- 8.15.1: 22
- 8.15.0: 13
- 8.15+rc1: 130

File: doc/sphinx/changes.rst [hunk 12]
```diff
@@ -11,6 +11,96 @@ Recent changes
...
+Jim Fehrle, Julien Forest, Emilio Jesús Gallego Arias, Gaëtan Gilbert,
+Georges Gonthier, Benjamin Grégoire, Jason Gross, Hugo Herbelin,
+Vincent Laporte, Olivier Laurent, Assia Mahboubi, Kenji Maillard,
+Guillaume Melquiond, Pierre-Marie Pédrot, Clément Pit-Claudel, Pierre Roux,
+Kazuhiko Sakaguchi, Vincent Semeria, Michael Soegtrop, Arnaud Spiwack,
+Matthieu Sozeau, Enrico Tassi, Laurent Théry, Anton Trunov, Li-yao Xia
+and Théo Zimmermann. See the `Coq Team face book <https://coq.inria.fr/coq-team.html>`_
+page for more details.
+
+The 57 contributors to the 8.16 versions are Tanaka Akira, Frédéric Besson, Martin Bodin, Ana Borges,
```
[the remark this answers is not in the document]
Comment (ana-borges) [reply doc/sphinx/changes.rst:80]: You can use "Ana Borges" everywhere. Thanks for the ping.

**Comment by mattam82** [comment]
I fixed the closed issues counts where I could

**Review by stop-cran — APPROVED** [review]

**Comment by ppedrot** [comment]
@coq/doc-maintainers Can some maintainer assign and merge? This seems ready and is needed for the 8.16 release.

**Review by Zimmi48 — APPROVED** [review]
@coqbot merge now

**Comment by Zimmi48** [comment]
@coqbot merge now

File: .mailmap [hunk 13]
```diff
@@ -28,6 +28,7 @@ Siddharth Bhat <siddu.druid@gmail.com>             Siddharth <siddu.druid@gmail.
 Lasse Blaauwbroek <lasse@blaauwbroek.eu>           Lasse Blaauwbroek <lasse@lasse-work.localdomain>
 Martin Bodin <martin.bodin@ens-lyon.org>           Martin Bodin <mbodin@ic.ac.uk>
 Ana Borges <ana.agvb@gmail.com>                    Ana <ana.agvb@gmail.com>
+Ana Borges <ana.agvb@gmail.com>                    ana-borges <ana-borges@users.noreply.github.com>
```
Comment (jfehrle) [inline .mailmap:31]: Duplicate entry

**Pull request merged**